### PR TITLE
Add an attribute use_distro_version

### PR DESCRIPTION
**Pull request by wenchma**
Added the attribute to support user using local repo.

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 maintainer_email  'cookbooks@opscode.com'
 license           'Apache 2.0'
 description       'Installs erlang, optionally install GUI tools.'
-version           '1.5.6'
+version           '1.5.7'
```
Comment (cmluciano) [inline metadata.rb]: Please remove the version bump. Oftentimes many PRs will be combined and released at once.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@ erlang Cookbook CHANGELOG
 =========================
 This file is used to list changes made in each version of the erlang cookbook.
 
+v1.5.7 (2014-09-11)
+-------------------
+Add an attribute use_distro_version for using local repo.
+
```
Comment (cmluciano) [inline CHANGELOG.md]: Please remove the version bump. Oftentimes many PRs will be combined and released at once.

File: README.md [hunk 3]
```diff
@@ -32,6 +32,7 @@ Attributes
   Erlang.
 * `node['erlang']['install_method']` - Erlang installation method
   ("package", "source", or "esl" (for Erlang Solutions packages)).
+* `node['erlang']['use_distro_version']` - Whether to use distro version erlang packages to install.
```
Comment (cmluciano) [inline README.md]: Can you adjust this to be a little clearer? Maybe something like "Use distribution-provided erlang packages"

**Comment by jtimberman** [comment]
Thank you for the pull request.

It appears the main change in this pull request is to make inclusion of the Erlang Solutions optional by an attribute.

In #29 (commit e56795001286ed54ec37d5e64f0a1d5d1cb7037e), I removed the recipe from `package`, because it wasn't intended that Erlang Solutions is used anywhere but the `esl` recipe.

As that commit makes this rather moot, I'll close it. Thank you though!

**Pull request closed without merging**